toolkit/tools/imagegen/installutils/installutils.go

```
@@ -843,21 +861,27 @@ func addUsers(installChroot *safechroot.Chroot, users []configuration.User) (err
 	// If no root entry was specified in the config file, never expire the root password
 	if !rootUserAdded {
 		logger.Log.Debugf("No root user entry found in config file. Setting root password to never expire.")
+
+		// Ignore updating if there is no shadow file to update
+		if exists, _ := file.PathExists(shadowFile); !exists {
```

>_ [](start = 13, length = 1)

I think we shouldn't ignore error even for simple functions.